extensions/amp-carousel/0.1/test/test-slides.js

```diff
@@ -918,18 +918,12 @@ describe('empty Slides functional', () => {
     expect(() => {
       slides.buildCallback();
     }).to.throw(/should have at least 1 slide/);
-    expect(() => {
-      slides.layoutCallback();
-    }).to.not.throw();
   });
```

Comment:
We might not need this test at all because the previous one already tested the `buildCallback`